Add instructions for using udev to manage GPU access (#325)
ROCm/rocm-install-on-linux: docs/install/prerequisites.rst

.. _group_permissions:

Configuring permissions for GPU access
================================================================

There are two primary methods to configure GPU access for ROCm: group membership or
udev rules. Each method has its own advantages, and the choice depends on your 
specific requirements and system management preferences.

> Using Group Membership

Reviewer: ```suggestion
Using group membership
```

The ``video`` and ``render`` groups are system groups in Linux used to manage access 
to graphics hardware and related functionality. Traditionally, the ``video`` group is used 
to control access to video devices, including graphics cards and video capture devices. 
The ``render`` group is more recent and specifically controls access to GPU rendering capabilities 
through Direct Rendering Manager (DRM) render nodes.

1. To check the groups in your system, issue the following command:

   .. code-block:: shell

       groups

2. Add yourself to the ``video`` and ``render`` groups:

   .. code-block:: shell

      sudo usermod -a -G video,render $LOGNAME

3. Optionally, add other users to the ``video`` and ``render`` groups:

   .. code-block:: shell

      sudo usermod -a -G video,render user1
      sudo usermod -a -G video,render user2

4. To add all future users to the render and video groups by default, run the following commands:

   .. code-block:: shell

      echo 'ADD_EXTRA_GROUPS=1' | sudo tee -a /etc/adduser.conf
      echo 'EXTRA_GROUPS=video' | sudo tee -a /etc/adduser.conf
      echo 'EXTRA_GROUPS=render' | sudo tee -a /etc/adduser.conf

> Using udev Rules

Reviewer: ```suggestion
Using udev rules
```

--------------------------------------------------------------------

> Udev rules offer a flexible way to manage device permissions. They apply system-wide, can be 

Reviewer: ```suggestion
A flexible way to manage device permissions is to use udev rules. They apply system-wide, can be 
```
@amd-jnovotny or feel free to make a suggestion to avoid double "Udev" entry in wordlist.  Or should we put udev in double-ticks throughout document?  ``udev``?

Reviewer 2: Good suggestion. I like the idea of rewording the sentence to avoid capitalizing udev. I think the actual utility is udevadm, so we technically don't want it in quotes. We could also potentially use the imperative voice here, for example, "Manage device permissions in a flexible manner using udev rules". @AMD-melliott feel free to rework this according to your preference.

Author: lgtm
